korrekt/src/circuit_analyzer/halo2_proofs_libs.rs

```diff
@@ -1,11 +1,27 @@
+/// ZCASH
+#[cfg(feature = "use_zcash_halo2_proofs")]
+pub use zcash_halo2_proofs::{
+    dev::MockProver,
+    circuit::{self, Value,Cell},
+    dev::{CellValue, Region},
+    plonk::{Advice, Any, Column, Expression, Selector,permutation, Assigned, Assignment, Circuit, ConstraintSystem, Error,
```

Comment:
Nit: permutation is not capitalized, and doesn't have a space. Does this affect its use later? (probably only the capitalization does.

Reply:
the `permutation` is imported from external halo2_proof, so we can't change the capitalization. Missing space added.